### PR TITLE
SAM-3299 - Show excepted due date on noLateSubmission screen

### PR DESCRIPTION
**Pull request by bgarciaentornos**

### CODE REVIEW

File: samigo/samigo-app/src/webapp/jsf/delivery/noLateSubmission.jsp [hunk 1]
```diff
@@ -43,10 +43,7 @@
   <h:outputText value="#{deliveryMessages.no_late_submission_url}" rendered="#{delivery.actionString=='takeAssessmentViaUrl'}"/>
   <p></p>
   <h:panelGroup  rendered="#{delivery.publishedAssessment.assessmentAccessControl.dueDate!=null}">
-    <h:outputLabel value="#{deliveryMessages.due_date}: " />
-	<h:outputText value="#{delivery.publishedAssessment.assessmentAccessControl.dueDate}" >
-		<f:convertDateTime dateStyle="medium" type="both" locale="#{UserLocale.locale}" timeZone="#{UserTimeZone.userTimeZone}"/>
-	</h:outputText>
+    <h:outputLabel value="#{deliveryMessages.due_date}: #{delivery.dayDueDateString}." />
```
Comment (ottenhoff) [inline samigo/samigo-app/src/webapp/jsf/delivery/noLateSubmission.jsp]: Is this dayDueDateString still localized for the user's preferred time zone?

Comment (bgarciaentornos) [reply samigo/samigo-app/src/webapp/jsf/delivery/noLateSubmission.jsp]: Yeah, it uses the TimeUtil class for getting the right timezone and I think it also checks the locale for the pattern.

**Comment by jonespm** [comment]
This original fix was for https://jira.sakaiproject.org/browse/SAM-2356 #1301

I think it was a problem when the assessment was linked to lessons. You should verify the test plan there to make sure that this still works and doesn't cause another regression.

**Comment by bgarciaentornos** [comment]
SAM-2356 verified and added code for retractDate case.

**Pull request merged**